feathr_project/feathr/utils/platform.py

```diff
@@ -33,13 +38,21 @@ def is_databricks() -> bool:
     Returns:
         bool: True if the module is running on Databricks notebook, False otherwise.
     """
-    try:
-        if str(Path(".").resolve()) == "/databricks/driver":
-            return True
-        else:
-            return False
-    except NameError:
+    # Note, this is a hacky way to check if the code is running on Databricks.
+    if str(Path(".").resolve()) == "/databricks/driver":
```

Comment:
Check DATABRICKS_RUNTIME_VERSION might be more stable

Reply:
this is great. I just changed the codes. Thank you!